extension/src/communications/from-extension-to-server.ts

```diff
@@ -1,7 +1,10 @@
 import { SyncIntent } from "../types";
 
 export enum MediaEventType {
+    NOP = "NOP",
```

Comment:
What does NOP stand for ? From what I read below, this is some kind of NotImplemented ?

Reply:
no operation